[2024-08-22T10:56:56.428+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-22T10:56:56.445+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_py_dependencies_v1.get_sk manual__2024-08-22T10:56:55.705543+00:00 [queued]>
[2024-08-22T10:56:56.454+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_py_dependencies_v1.get_sk manual__2024-08-22T10:56:55.705543+00:00 [queued]>
[2024-08-22T10:56:56.454+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 6
[2024-08-22T10:56:56.468+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): get_sk> on 2024-08-22 10:56:55.705543+00:00
[2024-08-22T10:56:56.477+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=256) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-08-22T10:56:56.479+0000] {standard_task_runner.py:72} INFO - Started process 257 to run task
[2024-08-22T10:56:56.477+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_py_dependencies_v1', 'get_sk', 'manual__2024-08-22T10:56:55.705543+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_python_dependencies.py', '--cfg-path', '/tmp/tmpdabxfjqn']
[2024-08-22T10:56:56.480+0000] {standard_task_runner.py:105} INFO - Job 3: Subtask get_sk
[2024-08-22T10:56:56.527+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_py_dependencies_v1.get_sk manual__2024-08-22T10:56:55.705543+00:00 [running]> on host 42e4ce4b26cd
[2024-08-22T10:56:56.608+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='matea' AIRFLOW_CTX_DAG_ID='dag_with_py_dependencies_v1' AIRFLOW_CTX_TASK_ID='get_sk' AIRFLOW_CTX_EXECUTION_DATE='2024-08-22T10:56:55.705543+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-22T10:56:55.705543+00:00'
[2024-08-22T10:56:56.609+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-22T10:56:57.308+0000] {logging_mixin.py:190} INFO - 1.5.0
[2024-08-22T10:56:57.309+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-08-22T10:56:57.316+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-22T10:56:57.317+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_py_dependencies_v1, task_id=get_sk, run_id=manual__2024-08-22T10:56:55.705543+00:00, execution_date=20240822T105655, start_date=20240822T105656, end_date=20240822T105657
[2024-08-22T10:56:57.363+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-22T10:56:57.388+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-22T10:56:57.390+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
